src/JackTripWorker.cpp

```diff
@@ -154,19 +129,20 @@ void JackTripWorker::run()
         //        qDebug() << "is WAIR?" <<  tmp ;
         qDebug() << "mNumNetRevChans" << mNumNetRevChans;
 
-        JackTrip jacktrip(JackTrip::SERVERPINGSERVER, JackTrip::UDP, mNumChans,
+        JackTrip jacktrip(JackTrip::SERVERPINGSERVER, JackTrip::UDP, 1, 1,
```

Comment:
Maybe the number of channels have to be set here. Or the arguments have to be removed.